packages/terra-form/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ Unreleased
 ### Added
 * Add missing props-table
 
+### Breaking Changes
+
+* Select and SelectField no longer accepts a list of strings, but a list of objects to set their value/display
```

Comment:
Is it worth keeping this passive by leaving the `choices` prop as-is and adding a `options` prop for this functionality. cc/ @mjhenkes @mhemesath